dev/raphael.svg.js

```diff
@@ -142,7 +142,7 @@
             }
         }
         $(o, {
-            fill: "url('" + document.location.origin + document.location.pathname + "#" + id + "')",
+            fill: "url('" + document.location.href+ "#" + id + "')",
```

Comment:
It fails when `document.location.hash` is not empty, see https://github.com/DmitryBaranovskiy/raphael/issues/933
Fixed with https://github.com/DmitryBaranovskiy/raphael/commit/f5d909be1b38c7afe18b9dd399c07759fc11a814.